benchmark/_test-double-benchmarker.js

```diff
@@ -18,12 +23,23 @@ function request(res) {
       run();
     } else {
       console.log(JSON.stringify({ throughput }));
+      if (client) {
+        client.destroy();
+      }
     }
   });
 }
 
 function run() {
-  http.get(url, request);
+  if (http.get) {
+    http.get(url, request);
+  } else {
+    const client = http.connect(url);
+    client.on('error', (e) => { throw e; });
+    const req = client.request({ ':path': '/' });
+    request(req, client);
+    req.end();
```

Comment:
nit: `':path'` defaults to '/' and, when using a `GET` request, the `req.end()` is unnecessary because http2 will automatically `end()`... so this can be simplified to:

```js
request(client.request(), client);
```

Reply:
Great, thanks, will change.